jvector-tests/src/test/java/io/github/jbellis/jvector/graph/Test2DThreshold.java

```diff
@@ -84,19 +84,71 @@ public void testThreshold() throws IOException {
 
     }
 
+    @Test
+    public void testThresholdHaversine() throws IOException {
```

Comment:
This test is failing. Not sure why yet.

Reply:
It was failing because I used the `biasedRandomFloat` method, which generated many vectors near the boundaries.